Vagrantfile

```diff
@@ -100,7 +105,11 @@ Vagrant.configure("2") do |config|
 
 	# Help the user out the first time they provision
 	config.vm.provision :shell do |shell|
-		shell.path = "puppet/postprovision.sh"
+		if CONF['_mode'] == "base"
+			shell.path = "puppet/preparebox.sh"
+		else
+			shell.path = "puppet/postprovision.sh"
+		end
```

Comment:
So right now, `base` will run `preparebox`, which means it's only really useful for building the base box. Since we offer this as a config option, should we distinguish between the "building the base box" and the "just using the base box instead of a custom box" use cases?